docker/install/install_python.sh

```
@@ -25,4 +25,4 @@ set -o pipefail
 wget -O Miniforge3.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
 bash Miniforge3.sh -b -p $1
 
-$1/bin/conda create -n $2 python=3.12
+$1/bin/conda create -n $2 python=3.12 pip
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)